mod/wikirate/set/type_plus_right/campaign/participant.rb

```diff
@@ -0,0 +1,27 @@
+
+
+# this is pretending as a pointer card
+def item_names args={}
+  initiative_name = cardname.left
+  # find all participant and add to the pointer card
+  editors = Card.search editor_of: { left: initiative_name }, return: "name"
+  editors += Card.search editor_of: { 
+                          left: {
+                            found_by: "#{initiative_name}+project" 
+                          } 
+                        },
+                        return: "name"
+  editors += Card.search editor_of: {
+                          right_plus: [ 
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Trailing whitespace detected.